actinia_stac_plugin/apidocs/stac_collections_docs.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Copyright (c) 2018-2021 mundialis GmbH & Co. KG
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+Documentation objects for GRASS modules and actinia modules api endpoints
+"""
+
+__author__ = "Carmen Tawalika"
+__copyright__ = "2018-2021 mundialis GmbH & Co. KG"
+__license__ = "Apache-2.0"
+
+from actinia_stac_plugin.apidocs.schemas import AddCollectionModel
+
+null = "null"
+
+
+staccollection_get_docs = {
+    "tags": ["STAC"],
+    "description": "Get a list of STAC collections" "Minimum required user role: user.",
+    "parameters": [],
+    "responses": {
+        "200": {"description": "This response returns a list of STAC collections"}
+    },
+}
+
+staccollection_post_docs = {
+    "tags": ["STAC"],
+    "description": "Add a new STAC collection to the user instance",
+    "parameters": [
+        {
+            "in": "body",
+            "name": "Adding Collection",
+            "type": "object",
+            "description": "the Instance ID where the Collection will be storaged",
+            "schema": AddCollectionModel,
+            "required": True,
+        }
```

Comment:
I am not 100% sure if this way of describing the API fits. I personally would have listed all two parameters here explicitly instead of using the AddCollectionModel because it might look like the user should provide a json like this:
```
"Adding Collection": {
        "stac_instance_id": "example1",
        "stac_url":  "example2"
}
```
But I haven't tried it out yet

Reply:
I tried out, but the documentation established that if the parameter "in" is body, a Schema is required. I tried to look for a way around, but I did not find it.

Reply:
I delete the classes and defined the schema directly there. See commit [a39d047](https://github.com/mundialis/actinia-stac-plugin/pull/29/commits/a39d04701bedb36fc4b436c190ef8ee39bb25081)

![image](https://user-images.githubusercontent.com/35478550/140072931-731ddf3f-040c-421c-9a8d-bb08db2f3b59.png)

Reply:
Ok fine, thanks. I confused something and it was ok before - sorry for this. But to be honest I like it better now when it is included directly as it has only one/two parameters and is not a complex schema.